openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
+
+        Parameters
+        ----------
+        bc : iterable of {'reflective', 'periodic', or 'vacuum'}
+            Boundary conditions for each of the six faces of a parallelopiped:
+            North, East, South, West, Up, and Down.
```

Comment:
There seems to be several ways to describe the six surfaces of a rectangle in OpenMC:

    x-min, x-max, y-min, y-max, z-min , z-max
    Left , Right, Back , Front, Bottom, Top
    East , West , South, North, Down  , Up        
    
I personally prefer using min and max because it leaves no ambiguity as to what dimension and direction the surface applies to while the others leave some ambiguity. For instance, the picture below indicates what I would think of the surfaces as. The front surface would be the y-min surface in cartesian coordinates, but OpenMC treats it as a y-max surface. This is more of personal preference, but my vote would be to refer to the surfaces with dimension + min/max.

![cube-4](https://cloud.githubusercontent.com/assets/1244537/17457161/51063978-5bbd-11e6-862f-18bdd8067c1f.jpg)